### PR TITLE
Expose Git SHA to window

### PR DESCRIPTION
**Pull request by kbariotis**
https://github.com/excalidraw/excalidraw/issues/1665

### CODE REVIEW

File: src/react-app-env.d.ts [hunk 1]
```diff
@@ -1 +1,5 @@
 /// <reference types="react-scripts" />
+
+interface Window {
+  __EXCALIDRAW_SHA__: string;
+}
```
Comment (dwelle) [inline src/react-app-env.d.ts:5]: I'd put this in `src/global.d.ts` if anything. Doesn't really belong here.

**Comment by dwelle** [comment]
Ok, wasn't aware Vercel injects this. Should have checked what we're doing with Sentry. Thanks!

**Comment by dwelle** [comment]
Mhm, I wonder where the commit SHA for e.g. the latest deployment is coming from: `4bd5b74d099c95f63b0f53f5500b47e28b981b5f` --- it doesn't refer to any commit in the repo that I can find. As such it's not very useful.

So I thought it maybe refers to the build, not repo commit, but [according to docs](https://vercel.com/docs/v2/git-integrations/vercel-for-github) it shouldn't:

> The GitHub SHA of the commit the deployment was triggered by.

**Comment by kbariotis** [comment]
@dwelle https://github.com/excalidraw/excalidraw/commit/4bd5b74d099c95f63b0f53f5500b47e28b981b5f ?

**Comment by dwelle** [comment]
Ok, but that's not on `master`.. vercel reuses latest relevant releases instead of building from latest master commit?

**Comment by kbariotis** [comment]
It was the last commit of a branch that ended up being merged on master: https://github.com/excalidraw/excalidraw/pull/1753/commits

Maybe Vercel is using that as the deployment commit?

**Comment by dwelle** [comment]
Seems like it, yea.

**Pull request merged**